### PR TITLE
[Report] Remove ReportBundle

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by Niiko** [comment]
:cry:

**Comment by DanPhyxius** [comment]
This would be a good feature for a lot of our customers.
Why remove it? :(

**Comment by Zales0123** [comment]
@Niiko @DanPhyxius I cry a lot about it also (pretty obvious why ;)), but there is sadly a good reason for that. For 1.0 we want to deliver only the most important features in Sylius (take a look at [latest docblock](http://sylius.org/blog/sylius-roadmap-update)). It's always a cost of maintenance with each bundle, but for now we should focus on delivering **really** stable, fully tested and reliable application. Nevertheless - I really hope *ReportBundle* will be back in some 1.* version (maybe with some new awesome features ;))

File: UPGRADE.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 UPGRADE
 =======
 
+## From 1.0.0-alpha to 1.0.0-beta
```
Comment (michalmarcinkowski) [inline UPGRADE.md]: We don't have `alpha` release yet. This should be in "From 0.19 to 1.0.0-alpha" section :)

**Comment by michalmarcinkowski** [comment]
Rebase needed.

**Comment by pjedrzejewski** [comment]
Thank you Grzesiu!

**Comment by lchrusciel** [comment]
ReportBundle R.I.P [*] :(

**Pull request merged**